src/_gettsim_tests/test_data/kindergeld/2021/3_children.yaml

```diff
@@ -0,0 +1,80 @@
+---
+info:
+  note: ''
+  source: ''
+inputs:
+  provided:
+    hh_id:
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+    p_id:
+      - 0
+      - 1
+      - 2
+      - 3
+      - 4
+    p_id_kindergeld_empf:
+      - -1
+      - -1
+      - 0
+      - 0
+      - 0
+    alter:
+      - 35
+      - 35
+      - 12
+      - 10
+      - 8
+    kind:
+      - false
+      - false
+      - true
+      - true
+      - true
+    arbeitsstunden_w:
+      - 0.0
+      - 0.0
+      - 0.0
+      - 0.0
+      - 0.0
+    in_ausbildung:
+      - false
+      - false
+      - true
+      - true
+      - true
+    bruttolohn_m:
+      - 20000.0
+      - 20000.0
+      - 0.0
+      - 0.0
+      - 0.0
+    _zu_verst_eink_ohne_kinderfreib_y_sn:
+      - 200000.0
+      - 200000.0
+      - 200000.0
+      - 200000.0
+      - 200000.0
+  assumed: {}
+outputs:
+  kindergeld_m:
+    - 663.0
+    - 0.0
+    - 0.0
+    - 0.0
+    - 0.0
+  _kindergeld_erstes_kind_m:
```

Comment:
Why would this be the same for all members of the household?

Reply:
Because this is a parameter that is the same for any individual given the policy date (as, for example, `minijob_grenze`)

Reply:
Is it worth testing that as an output then?

Reply:
Wanted to test it somewhere, but making it part of an "economic" test case is probably not great. Should I move that to `test_kindergeld.py`?

Reply:
I would be fine with testing it implicitly, but whatever you prefer!